src/browser/web-view/shell.js

```diff
@@ -51,7 +51,7 @@ const ZoomChanged =
   ({type: "ZoomChanged", zoomResult: result});
 
 const setZoom = (id, level) =>
-  Task.future(() => {
+  Task.create(resolve => {
```

Comment:
We need to keep a promise wrapper here in order to keep an async resolution behavior as it was prior to this change (See https://github.com/browserhtml/browserhtml/pull/1003#issuecomment-211648650 for more details)